### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by peekpt** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#133] [Q&A] [answered]
Now that's all installed an running with post installation,
how do I update EFI to: "Latest Update 16 June 2022 OpenCore 0.8.2" ?

**Comment by chris1111** [comment]
[See here](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/discussions/71#discussioncomment-692730)

**Reply by peekpt** [reply]
So I'll need to backup all my smc ids and the intlwm.kext I or any configs I added ?

**Comment by chris1111** [comment, marked as answer]
yes

**Reply by peekpt** [reply]
I cloned and ran the repo in Xcode.
I got and app with a build button.
I clicked build button and later got some folder with apps, one for make the install media other for the post install.
I cleaned my internal disk EFI folder and started and completed the post install.
I opened the Open Core configurator, mount the EFi and restore my serials and uuids and my added Kext for wifi.
rebooted and everything looks normal.

Now I got the feeling I'm on 0.8.0

**Reply by peekpt** [reply]
update I'm on 0.8.2 
```
# nvram 4D1FDA02-38C7-4A6A-9CC6-4BCCA8B30102:opencore-version
4D1FDA02-38C7-4A6A-9CC6-4BCCA8B30102:opencore-version	REL-082-2022-06-15
```
